styles/widgets/material/button.material.less

```diff
@@ -71,7 +68,6 @@
 }
 
 .dx-button-has-text {
-    min-width: @MATERIAL_BUTTON_MIN_WIDTH;
```

Comment:
It can change the default dxButton width

Reply:
fixed it

Reply:
> It can change the default dxButton width

fixed it